closed/autoconf/custom-hook.m4

```diff
@@ -649,16 +652,16 @@ AC_DEFUN([CONFIGURE_OPENSSL],
 ])
 
 # Create a tool wrapper for use by cmake.
-# Consists of a shell script which wraps commands with an invocation of fixpath.
-# OPENJ9_GENERATE_TOOL_WRAPER(<name_of_wrapper>, <command_to_call>)
+# Consists of a shell script which wraps commands with an invocation a wrapper command
```

Comment:
Something seems missing here, perhaps you meant to say 'invocation _of_ a wrapper command' (with a period).